spec/index.bs

```diff
@@ -1255,7 +1255,7 @@ The <dfn method for="GPUDevice">createBindGroupLayout(|descriptor|)</dfn> method
 <b>Validation Conditions</b>
 
 <dl dfn-for="createBindGroupLayout(descriptor)">
-If any of the following conditions are violated:
+<dfn>invalid layout</dfn>: If any of the following conditions are violated:
```

Comment:
sorry, this isn't what I meant. In the top-of-tree version of this spec there is already a definition for the word `[=invalid=]` which is general for all objects. You should use that to define what a "valid" GPUBindGroupLayout is.

Reply:
See https://github.com/gpuweb/gpuweb/pull/538#discussion_r366095535

Reply:
I didn't fetch the latest code, so I misunderstood your suggestions...